src/tslint.ts

```diff
@@ -86,6 +86,21 @@ class Linter {
     private containsRule(rules: Lint.RuleFailure[], rule: Lint.RuleFailure) {
         return rules.some((r) => r.equals(rule));
     }
+
+    private getRulesDirectories(): Array<string> {
+        const rulesDirectories: Array<string> = [];
+
+        if (this.options.rulesDirectory) {
+            if (typeof this.options.rulesDirectory === "string") {
+                rulesDirectories.push(this.getRelativePath(this.options.rulesDirectory as string));
+            } else {
+                const relativePathDirectories = (this.options.rulesDirectory as Array<string>).map(dir => this.getRelativePath(dir));
+                rulesDirectories.push(...relativePathDirectories);
```

Comment:
Done in commit https://github.com/palantir/tslint/commit/ee60a0cd1b858bd9555f21e9b4efdb501ebb570d.